### PR TITLE
fix red tests

### LINKED ISSUE #40: main tests are red
#### pulumi program is not downloading the correct provider plugin
```
Diagnostics:
  pulumi:providers:aws (default_4_38_1):
    error: no resource plugin 'aws-v4.38.1' found in the workspace or on your $PATH, install the plugin using `pulumi plugin install resource aws v4.38.1`
 
Resources:
    + 1 created

Duration: 1s

Unhandled exception. Pulumi.Automation.Commands.Exceptions.CommandException: code: 255
```

#### database migration examples rely on AMI that no longer exists
database migration tests fail with similar errors to such.

```
Diagnostics:
  pulumi:pulumi:Stack (database_migration-dev):
    error: update failed

  aws:rds:Cluster (db):
    error: 1 error occurred:
    	* error creating RDS cluster: InvalidParameterCombination: Cannot find version 5.7.mysql_aurora.2.03.2 for aurora-mysql
    	status code: 400, request id: 295f4c4d-3b43-427f-a3dd-bd9ef30d7c6c

Resources:
    + 3 created
```

#### typescript invalid syntax on types
node database migration CI error. appears to be caused by having an old version of typescript.

yarn run v1.22.19
$ tsc && node ./bin/index.js
Error: node_modules/@types/aws-lambda/trigger/cloudfront-request.d.ts(1,13): error TS1005: '=' expected.
Error: node_modules/@types/aws-lambda/trigger/cloudfront-request.d.ts(1,83): error TS1005: ';' expected.
Error: node_modules/@types/aws-lambda/trigger/cloudfront-request.d.ts(2,13): error TS1005: '=' expected.
Error: node_modules/@types/aws-lambda/trigger/cloudfront-request.d.ts(2,40): error TS1005: ';' expected.
error Command failed with exit code 2.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
Error: Process completed with exit code 2.

### Steps to reproduce

run CI

### Expected Behavior

CI passes

### Actual Behavior

doesn't pass

### Output of `pulumi about`

_No response_

### Additional context

_No response_

### Contributing

Vote on this issue by adding a 👍 reaction. 
To contribute a fix for this issue, leave a comment (and link to your pull request, if you've opened one already).

### PR DESCRIPTION
**Pull request by dixler**
the tests were red. opened an issue #40 describing the issues.

Fixes #40

### CODE REVIEW

**Comment by dixler** [comment]
dotnet-database-migration has a resource that uses an AMI that no longer exists.

```
Diagnostics:
  aws:rds:Cluster (db):
    error: 1 error occurred:
    	* error creating RDS cluster: InvalidParameterCombination: Cannot find version 5.7.mysql_aurora.2.03.2 for aurora-mysql
    	status code: 400, request id: 019c0c7c-1496-4307-9183-2603f562c7bd
```

**Comment by dixler** [comment]
node database migration CI error

```
yarn run v1.22.19
$ tsc && node ./bin/index.js
Error: node_modules/@types/aws-lambda/trigger/cloudfront-request.d.ts(1,13): error TS1005: '=' expected.
Error: node_modules/@types/aws-lambda/trigger/cloudfront-request.d.ts(1,83): error TS1005: ';' expected.
Error: node_modules/@types/aws-lambda/trigger/cloudfront-request.d.ts(2,13): error TS1005: '=' expected.
Error: node_modules/@types/aws-lambda/trigger/cloudfront-request.d.ts(2,40): error TS1005: ';' expected.
error Command failed with exit code 2.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
Error: Process completed with exit code 2.
```

File: python/local_program/aws-py-voting-app/requirements.txt [hunk 1]
```diff
@@ -1,3 +1,3 @@
 pulumi>=3.0.0,<4.0.0
 pulumi-aws>=4.0.0,<5.0.0
-pulumi-docker>=3.0.0,<4.0.0
+pulumi-docker==3.1.0
```
Comment (dixler) [inline python/local_program/aws-py-voting-app/requirements.txt:3]: pinned as per https://github.com/pulumi/pulumi-docker/issues/245

**Review by dixler — COMMENTED** [review]
Tests are green now

**Review by aq17 — APPROVED** [review]
LGTM!

**Pull request merged**